### PR TITLE
feat(exactly): ✨ add claimable rewards

### PR DESCRIPTION
**Pull request by itofarina**
## Description

Add claimable rewards for [Exactly](https://zapper.xyz/apps/exactly)

## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [x] (optional) As a contributor, my Ethereum address/ENS is: `dev.exa.eth`
- [x] (optional) As a contributor, my Twitter handle is: https://twitter.com/ExactlyProtocol

### CODE REVIEW

File: src/apps/exactly/common/exactly.rewards.contract-position-fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    @Inject(APP_TOOLKIT) protected readonly appToolkit: IAppToolkit,
+    @Inject(ExactlyContractFactory) protected readonly contractFactory: ExactlyContractFactory,
+    @Inject(ExactlyDefinitionsResolver) protected readonly definitionsResolver: ExactlyDefinitionsResolver,
+  ) {
+    super(appToolkit);
+  }
+
+  async getDefinitions({ multicall }: GetDefinitionsParams) {
+    const address = PREVIEWER_ADDRESSES[this.network];
+    if (!address) throw new Error(`missing previewer on ${this.network}`);
```
Comment (wpoulin) [inline src/apps/exactly/common/exactly.rewards.contract-position-fetcher.ts:45]: Line 44 and 45 won't be necessary as both lines are already in `ExactlyDefinitionsResolver.getDefinitions()`

**Review by wpoulin — COMMENTED** [review]
Does Ethereum also have claimable rewards ?

File: src/apps/exactly/common/exactly.rewards.contract-position-fetcher.ts (hunk 1 above)
Comment (itofarina) [reply src/apps/exactly/common/exactly.rewards.contract-position-fetcher.ts]: done, thanks!

**Comment by itofarina** [comment]
> Does Ethereum also have claimable rewards ?

thanks for reviewing @wpoulin!
not for the moment

**Review by wpoulin — APPROVED** [review]

**Pull request merged**